src/main/java/edu/illinois/library/cantaloupe/resource/iiif/v1/InformationResource.java

```diff
@@ -42,6 +46,21 @@ public Method[] getSupportedMethods() {
         return SUPPORTED_METHODS;
     }
 
+    @Override
+    protected final void doOPTIONS() {
```

Comment:
Is there a reason we can't put this method in `IIIFResource` and then we only need it one time rather than the three times?

Reply:
I remember asking this question to Mark, too, but unfortunately do not remember what his response was at the time.

Just looking at it myself, there are other things that extend IIIF1Resource/IIIF2Resource/IIIF3Resource (which extend IIIFResource), like: LandingResource, IdentifierResource, and ImageResource. My guess would be that Mark didn't feel the doOptions() implementation he added was appropriate for those other resources, so kept the scope more limited to the info resources (across all 3 versions).

Reply:
@jcoyne are you happy with this? Note the IIIF1Resource etc are the only info requests which need protecting so the other resources don't require auth.

Reply:
A deeper refactoring could happen so that the headers were only presented on appropriate objects, but that would involve a more significant refactoring of Cantaloupe itself, which seems more involved than our first goal of just getting caught up on the queued PRs.